components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1760,9 +1760,11 @@ type Workspace struct {
 	// during workspace creation. Once a base image has been built the information in here
 	// is superseded by baseImageNameResolved.
 	ImageSource interface{} `json:"imageSource,omitempty"`
-	OwnerID     string      `json:"ownerId,omitempty"`
-	Pinned      bool        `json:"pinned,omitempty"`
-	Shareable   bool        `json:"shareable,omitempty"`
+	// undefined means it is owned by the user (legacy mode, soon to be removed)
+	OrganizationId string `json:"organizationId,omitempty"`
```

Comment:
Is undefined a string value? Or is it empty? I'd expect that we never end up with the value `"undefined"` here as that's a NodeJS concept only and it should either be empty, or a `pointer` to string

Reply:
FWIW, we are very soon making the organizationId mandatory on workspace objects. They are already always existent so you can treat it like that already.